docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -69,6 +69,14 @@ You can also use Task Manager in {kib} to troubleshoot background tasks and proc
 * {kibana-ref}/task-manager-health-monitoring.html[Task Manager health monitoring]
 * {kibana-ref}/task-manager-troubleshooting.html[Task Manager troubleshooting]
 
+[float]
+[[troubleshoot-max-alerts]]
+==== Troubleshoot max alerts warning
+
+When a rule reaches the maximum number of alerts it can generate in a single rule execution, the following warning is displayed on the rule's details page and in the rule execution log: `This rule reached the maximum alert limit for the rule execution. Some alerts were not created.` 
+
+If you receive this warning, go to the rule's **Alerts** tab and examine the alerts for anything unexpected. Unexpected alerts might be created from data source issues or queries that are too broadly-scoped. To further reduce alert volume, you can also add <<add-exceptions,rule exceptions>> or <<alert-suppression,suppress alerts>>. 
```

Comment:
```suggestion
If you receive this warning, go to the rule's **Alerts** tab and look for anything unexpected. Unexpected alerts might be created from data source issues or queries that are too broadly-scoped. To further reduce alert volume, you can also add <<add-exceptions,rule exceptions>> or <<alert-suppression,suppress alerts>>. 
```

Reply:
While "look" totally works, I'd prefer "check" since it's less ableist. :)